README.txt

```diff
@@ -47,49 +47,27 @@ This plugin is, like all of the Unlock code, [open source](https://github.com/un
 
 = Local Development =
 
-To setup local development environment for the plugin using [Lando](https://docs.lando.dev/basics/installation.html) local dev environment, simply clone the repo from [GitHub](https://github.com/unlock-protocol/unlock-wordpress-plugin) and write
+To set up the local development environment for the plugin using [LocalWP](https://localwp.com/) local dev environment:
 
-    lando start
+1. Simply download and install the LocalWP.
 
-About live editing, from the assets directory, when editing the main.scss, you can run:
+2. Set up a local wp website with it and run it in your browser.
 
-    lando npm run start
+3. Download this Unlock WP Plugin repo as a zip file.
```

Comment:
This part is not very clear to me @SolomonFoskaay , should it be downloaded from Github?

I think it is useful to be able to deploy from files changed locally?

Reply:
Yeah, the localwp is similar to the live wp just run locally. 
So, to use the plugin, you need to download it as zip file from the repo (simply means download the repo as zip instead of cloning it) and upload it as a normal Wordpress which will upload it to the localwp dev plugin folder.

Then you can edit it using text editors like VScode, any changes made are saved and instantly reflect on the localwp website. There is no special build process like Lando or Docker. It is a native Wordpress local development that works across major OS like Windows, MacOS and different Linux distros. making it easier to access with no other setup than the app installation itself.

Reply:
> This part is not very clear to me @SolomonFoskaay , should it be downloaded from Github?
> 
> I think it is useful to be able to deploy from files changed locally?

@julien51 Yeah, the localwp is similar to the live wp just run locally. 
So, to use the plugin, you need to download it as zip file from the repo (simply means download the repo as zip instead of cloning it) and upload it as a normal Wordpress which will upload it to the localwp dev plugin folder.

Then you can edit it using text editors like VScode, any changes made are saved and instantly reflect on the localwp website. There is no special build process like Lando or Docker. It is a native Wordpress local development that works across major OS like Windows, MacOS and different Linux distros. making it easier to access with no other setup than the app installation itself.